packages/xo-web/src/common/intl/messages.js

```diff
@@ -14,6 +14,12 @@ const messages = {
   errorNoSuchItem: 'no such item',
   errorUnknownItem: 'unknown {type}',
   memoryFree: '{memoryFree} RAM free',
+  date: 'Date',
+  notifications: 'Notifications',
```

Comment:
_Notifications_ or _messages_?

Reply:
Good question. "Notifications" sounds more usual to me but I don't have a strong opinion on this.

@Marcpezin? @olivierlambert?

Reply:
I have a preference for notifications which is pretty common in the app and soft industry.